configure.ac

```diff
@@ -53,7 +53,7 @@ AC_ARG_WITH(nopylint,
 
 [if test "$with_nopylint" = "no"]
 [then]
-    AC_PATH_PROG([PYLINT], [python3-pylint], [no])
+    AC_PATH_PROG([PYLINT], [pylint-3], [no])
```

Comment:
Please fix me if I'm wrong but wouldn't it work and be portable if we wrote something like:

```AC_PATH_PROGS([PYLINT], [pylint-3 python3-pylint], [no])```

Note that there is ```_PROGS``` rather than ```_PROG``` and the second argument is a blank-separated list which may contain more elements.